build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/miscTasks.ts

```diff
@@ -236,13 +236,19 @@ export class TypeValidationTask extends LeafWithFileStatDoneFileTask {
 	 */
 	protected async getInputFiles(): Promise<string[]> {
 		if (this.inputFiles === undefined) {
-			const { packageJsonPath: previousPackageJsonPath } = getTypeTestPreviousPackageDetails(
-				this.node.pkg,
-			);
-			this.inputFiles = [
-				path.join(this.node.pkg.directory, "package.json"),
-				previousPackageJsonPath,
-			];
+			this.inputFiles = [path.join(this.node.pkg.directory, "package.json")];
+			if (!(this.node.pkg.packageJson.typeValidation?.disabled === true)) {
```

Comment:
I facepalmed... I was _so close_ to getting this right the first time. Thanks for finding and fixing!

Reply:
Nit: you could just use !== true, right? Seems a little clearer to me.

Reply:
Never mind... it's too early for me to code review apparently. Ignore this lunacy. :)